common/src/main/java/com/wynntils/core/features/overlays/OverlayManager.java

```diff
@@ -28,18 +28,20 @@ public class OverlayManager {
 
     public static void registerOverlay(Overlay overlay, OverlayInfo overlayInfo) {
         overlayInfoMap.put(overlay, overlayInfo);
-
-        if (overlayInfo.enabled()) {
-            enabledOverlays.add(overlay);
-        }
     }
 
     public static void disableOverlays(List<Overlay> overlays) {
         enabledOverlays.removeIf(overlays::contains);
     }
 
-    public static void enableOverlays(List<Overlay> overlays) {
-        enabledOverlays.addAll(overlays);
+    public static void enableOverlays(List<Overlay> overlays, boolean ignoreState) {
```

Comment:
Is this called with true? I can't find it.

Reply:
It is not right now, but we will when I add Overlay config options so the user can enable/disable them (by commands at least atm).